app/models/customer.py

```diff
@@ -0,0 +1,21 @@
+from flask import current_app
+from app import db
+
+class Customer(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.String)
+    phone_num = db.Column(db.String)
+    register_at = db.Column(db.DateTime, nullable=True)
+    videos_checked_out_count = db.Column(db.Integer, default=0)
```

Comment:
Instead of this being a column you can use the rentals to count that.  

To do that it's helpful to add a relationship:
```
rentals = db.relationship('Rental', back_populates='customer', lazy=True)
```
Once you have the relationship the `videos_checked_out_count` is just the length of the `rentals` that haven't been checked in:

```
"videos_checked_out_count": len(rental for rental in self.rentals if not rental.outstanding)
```